### DISCUSSION TITLE
The proposed roi is larger than the image.

### DISCUSSION THREAD

**Discussion by zhouzhenghong-gt** [Project-MONAI/MONAI#2036] [Q&A]
Hello! This is a very good work, but I have encountered some problems. 

When I tried to apply monai to a new data set (I used the "spleen_segmentation_3d" code), an error occurred: "The proposed roi is larger than the image" in line "for batch_data in train_loader:". I want to know what might be the reason?

**Comment by Nic-Ma** [comment]
Hi @Zzzzzzzzzzh ,

This may because the random crop size of ROI is larger than actual image size.

Thanks.

**Reply by Nic-Ma** [reply]
Actually, we already received many feedback from the internal team and MONAI users that don't understand what this error message means. Let me try to enhance it.

Thanks.

**Reply by Nic-Ma** [reply]
PR #2038 address this issue.
Thanks.
